scss/style.scss

```diff
@@ -9,8 +9,54 @@
   font-style: normal;
 }
 
-body {
-  // background-image: linear-gradient(bottom, rgb(254, 133, 107) 24%, rgb(35, 171, 17) 62%);
+// body {
+//   background-image: linear-gradient(bottom, rgb(254, 133, 107) 24%, rgb(35, 171, 17) 62%);
+// }
+
+.success-message-p {
+  background-color: $blue;
+  padding: 10px 15px;
+  color: $white;
+  box-shadow: 0 0 5px $white;
+  display: none;
+}
+
+.spinner-section {
+  display: none;
+  justify-content: center;
+  padding-top: 10px;
+}
```

Comment:
one of the purposes of variables is to reuse one value in many places. as we working a lot with spacing and giving margin in styling. so we can define variables for our spacing sizes and define utility classes for that. so please kindly define variables for your spacing properties and reuse it where it's necessary. 👍🏻